tests/lib/schema_validation_test.py

```diff
@@ -170,6 +170,10 @@ def test_valid_without_auto_neg_and_missing_speed(self, default_data):
 
         libnmstate.validator.validate(default_data)
 
+    def test_valid_with_sriov_total_vfs(self, default_data):
+        default_data[Interface.KEY][0].update({'sr-iov': {'total-vfs': 5}})
```

Comment:
Please use the constants here for `sr-iov` and `total-vfs`.

Reply:
Done! Thanks!